packages/core/src/sagas/markAllAsSpokenOnStopSpeakActivitySaga.js

```diff
@@ -0,0 +1,20 @@
+import {
+  put,
+  select,
+  takeEvery
+} from 'redux-saga/effects';
+
+import { STOP_SPEAKING_ACTIVITY } from '../actions/stopSpeakingActivity';
+import markActivity from '../actions/markActivity';
+
+export default function* () {
+  yield takeEvery(STOP_SPEAKING_ACTIVITY, function* () {
+    const { activities } = yield select();
```

Comment:
Added new `speakingActivity` predicate to "definitions" directory.

Instead of selector, we have "definitions" set up for this purpose. IMO, definition is better than selector because we can select/query by combining multiple definitions (and/or).